packages/shared/src/components/post/BasePostContent.tsx

```diff
@@ -51,6 +59,11 @@ export function BasePostContent({
       {customNavigation ?? (
         <PostNavigation {...navigationProps} className={className.navigation} />
       )}
+      {withPillTypes.includes(post?.type) && (
+        <span className="py-2 px-3 mt-6 font-bold capitalize rounded-8 bg-[#BF29F014] text-theme-color-cabbage typo-footnote w-fit">
```

Comment:
The usage of `bg-[#BF29F014]` is temporary, I have already raised a discussion with the design team.

Reply:
Bg-cabbage-float i think will do,
There should be some examples for the exact classname.

Also should this always show or only v1 feed?

Reply:
Was not available unfortunately. The available one was `overlay-float` which doesn't produce any color.

Reply:
This one is the one:
`bg-theme-overlay-float-cabbage`

Overlay means the .50 version and float means 8%